### ISSUE TITLE
When there is no “using” statement in the class it is introducing a blank new line after namespace causing SA1505

### ISSUE THREAD

**Issue by Jebarson** [dotnet/roslyn#49497]
**Version Used**: VS 16.8.1

**Steps to Reproduce**:

1.  Create a project template with a class that has no using statement
2.  Create a project using the template

**Expected Behavior**:
The code is generated as done in the template.

**Actual Behavior**:
The code generated has a new empty line after the namespace brace. This introduces SA1505

**Comment by CyrusNajmabadi** [comment]
Wouldn't this be expected as it would be virtually certain that users would be adding something inside the namespace?  So erring toward the uncommon case where nothing was added wouldn't be beneficial.

> This introduces SA1505

This is an external analyzer.  Their decisions aren't really relevant here to the defaults in an IDE, esp. ones around how people will actually commonly type.

**Comment by Jebarson** [comment]
Nope that would a very vague assumption. There are plenty of situation from abstract classes to classes with constants that will never need even a system namespace.

**Comment by CyrusNajmabadi** [comment]
> The code generated has a new empty line after the namespace brace. 

Can you give a screenshot of what you mean?

**Comment by CyrusNajmabadi** [comment]
> There are plenty of situation from abstract classes to classes with constants that will never need even a system namespace.

Right.  But we may design around the common case, not the uncommon case.

**Comment by Jebarson** [comment]
![image](https://user-images.githubusercontent.com/10510974/99707150-eb65ce00-2a50-11eb-8fe7-b7c9ec8583b2.png)

**Comment by CyrusNajmabadi** [comment]
Oh, i misunderstood your case then.  i thought you were referring to the template generating:

```
namespace N
{

}
```

I would not consider that a bug.  I do consider what you're seeing to be undesirable.

**Comment by Jebarson** [comment]
Regardless whatever you call, it causes SA1505 and now my VS template creates code with a compilation error. Yes it is a mandate for us to mark all warnings as error.

**Comment by CyrusNajmabadi** [comment]
> it causes SA1505

As i mentioned above, this is an external analyzer.  It is not a goal of roslyn to not cause satisfy external analyzers.  

> And remember this was not always the case.

Yes.  Our behavior chnages over time.  We do not give a guarantee that we'll always operate in the exact same fashoin in the IDE.

That said, as i mentioned above, in this particular case, i think we would want this new line to not be there.  If you contributed a PR there, we would likely take it.  Thanks!
